### PR TITLE
i18n(es): update `i18n`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes # N/A 
- What does this PR change? Give us a brief description.
Updates `i18n` based on these commit:
  - ffe27fcbf9793a6634891aa72babfaf5eaeff204
  - ca0678ca556d739bda9648edc1b79c764fdea851

### CODE REVIEW

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 1]
```diff
@@ -142,6 +142,34 @@ Starlight espera que crees páginas equivalentes en todos tus idiomas. Por ejemp
...
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+-			title: 'Mi Documentación',
++			title: {
++				en: 'Mi Documentación',
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/i18n.mdx:160]:
```suggestion
+				es: 'Mi Documentación',
```

Small tweak to ensure the config is valid

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except for a small missing change, everything looks good to me.

**Comment by dreyfus92** [comment]
Thanks again @HiDeoo! 😁

**Review by HiDeoo — APPROVED** [review]
Looks perfect to me now, thanks a lot.

**Pull request merged**